src_new/util.js

```diff
@@ -913,3 +913,28 @@ exports.ajaxRequest = function(url, callback, data, options) {
 		refThis.log(error);
 	}
 };
+
+// Returns mediaTypes for adUnits which are sent to prebid
+exports.getMediaTypeObject = function(nativeConfig, sizes, currentSlot){
+	var mediaTypeObject = {};
```

Comment:
please mention on confluence that we are not considering size based kgp in first phase as Native creatives are not restricted to sizes of adslot

Reply:
Done Update the confluence